org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/preferences/BuilderConfigurationBlock.java

```diff
@@ -388,4 +393,8 @@ public String getPropertyPrefix() {
 		return PROPERTY_PREFIX;
 	}
 
+	@Override
+	protected Predicate<Entry<String, ValueDifference<String>>> getAffectsBuilderPredicate() {
```

Comment:
OptionsConfigurationBlock is api so i wonder if we should make this method api too

Reply:
It should become API, yes. But I'd prefer a better type signature and easier means to customize it.